static/openapi/eds-spec.json

```diff
@@ -0,0 +1,377 @@
+        "tags": [
+          "Event Subscription"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "stream",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The name of the stream to subscribe to."
+          },
+          {
+            "in": "query",
+            "name": "type",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The type of events to subscribe to."
+          },
+          {
+            "in": "query",
+            "name": "filters",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "An optional query parameter to filter events."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful response.",
+            "headers": {
+              "X-PhotonIQ-ES": {
+                "description": "Epoch time header.",
+                "schema": {
+                  "type": "string",
+                  "description": "Epoch time in seconds."
+                }
+              },
+              "X-PhotonIQ-CustomerId": {
+                "description": "Customer Id header.",
+                "schema": {
+                  "type": "string",
+                  "description": "Customer Id that includes the tenant and fabric name.."
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "error": {
+                      "type": "string",
+                      "description": "Error message."
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/es/v1/metrics": {
+      "get": {
+        "summary": "Metrics",
+        "description": "Retrieve metrics data within a specified time range.",
+        "tags": [
+          "EDS Metrics"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "start",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The start timestamp for the metrics data range."
```

Comment:
what format is this timestamp?  Examples how others did it:
https://photoniqall.eng.macrometa.io/docs/ds (look at GET /api/ds/v1/metrics)
https://photoniqall.eng.macrometa.io/api/est/api-docs/#/History/get_api_est_v1_history (look at GET /api/est/v1/history)

Reply:
It is UTC format. Added example.